components/PlayoffsBracket.tsx

```diff
@@ -58,7 +59,7 @@ function PlayoffsBracket({ data, league }: Props): JSX.Element {
       home: teamData?.find(team => team.id === serie.team2)?.colors.primary || '#BBB'
     };
     const awayTeam = {
-      id: serie.team1 || -1,
+      id: serie.team1 >= 0 ? serie.team1 : -1,
```

Comment:
not sure why it says `serie` instead of `series`, can we fix that rq and then i'll merge

Reply:
Good catch. I blame my non-nativeness for that one.
Fixed everywhere